jahsieAult/question5b.js

```diff
@@ -0,0 +1,120 @@
+// Error-detecting version
+
```

Comment:
5) You'll have a lot of problems here because the assertions are expecting the functions to have different names than you assigned them. They expect the function you've named `findRank` to be `rank`, for example. Also, try to avoid declaring global variables (like your `suit`, `rank`, and `id`) whenever possible--it can lead to some pretty confusing problems. Once you've got the assertions up and running, I'd be happy to go over any remaining questions with you; I think getting the assertions going will help you quite a bit.